shared/teams/new-team/wizard/create-channels.tsx

```diff
@@ -35,24 +43,31 @@ const CreateChannel = () => {
     setChannels([...channels])
   }
 
+  const filteredChannels = channels.filter(c => c.trim())
   const onContinue = () =>
-    dispatch(TeamsGen.createSetTeamWizardChannels({channels: channels.filter(c => !!c)}))
+    onSubmitChannels
+      ? onSubmitChannels(filteredChannels)
+      : dispatch(TeamsGen.createSetTeamWizardChannels({channels: filteredChannels}))
   const onBack = () => dispatch(nav.safeNavigateUpPayload())
   const onClose = () => dispatch(RouteTreeGen.createClearModals())
 
-  const numChannels = channels.filter(c => !!c.trim()).length
+  const numChannels = filteredChannels.length
   const continueLabel = numChannels
     ? `Continue with ${numChannels} ${pluralize('channel', numChannels)}`
     : 'Continue without channels'
```

Comment:
I don't feel like continuing without channels should be a valid operation in the existing team version of this. Maybe disable the button with 0, and with 1 or more make it "create N channels" instead of "continue with"?